### PR TITLE
Fix config values flow to modules

### LINKED ISSUE #7728: Configs values are not flowing to modules init
### Expected

When we pass module specific configuration to each module in `config.json` in `modules: { fee: { feeTokenID: '02000000' } }` it should be accessible within `init(moduleConfig)`

### Actual

When we pass module specific configuration to each module in `config.json` in `modules: { fee: { feeTokenID: '02000000' } }`
it doesn't come in `moduleConfig`

### Steps to reproduce

- Update `config.json` with `modules: { fee: { feeTokenID: '02000000' } }` and check if fee module has passed `feeTokenID` or not.

- Related issue https://github.com/LiskHQ/lisk-sdk/issues/7710

### Affected version

- `6.0.0-alpha.0`

### PR DESCRIPTION
**Pull request by ishantiw**
### What was the problem?

This PR resolves #7728 , #7738, #7739

### How was it solved?

- [Pass `ModuleConfig` to `stateMachine init`](https://github.com/LiskHQ/lisk-sdk/commit/59dae067d39ac5cb1cc87631c9a2bac7e5f76f20)
- [Expose and add endpoint to expose `multisigRegMsgSchema`](https://github.com/LiskHQ/lisk-sdk/commit/f79288bc20805df40372a220c911dfd61116f224)

### How was it tested?

- Run `dpos-mainchain` app and add in the config to test if module config is flowing to the modules or not
```js
{ modules: "fee": {
			"feeTokenID": "0200000000000000"
		}, }
```
- To test the endpoint, call `auth_getMultiSigRegMsgSchema`

### CODE REVIEW

File: framework/src/application.ts [hunk 1]
```diff
@@ -279,7 +279,12 @@ export class Application {
 				endpoints: this._rootEndpoints(),
 				events: [APP_EVENT_READY.replace('app_', ''), APP_EVENT_SHUTDOWN.replace('app_', '')],
 			});
-			await this._stateMachine.init(this.logger, this.config.genesis, this.config.modules);
+			await this._stateMachine.init(
+				this.logger,
+				this.config.genesis,
+				this.config.generator,
```
Comment (shuse2) [inline framework/src/application.ts:285]: generator config can be removed from state machine and  `init` interface for module

Comment (ishantiw) [reply framework/src/application.ts]: removed

**Review by shuse2 — APPROVED** [review]

File: framework/src/controller/channels/in_memory_channel.ts [hunk 2]
```diff
@@ -37,10 +38,12 @@ export class InMemoryChannel extends BaseChannel {
 		namespace: string,
 		events: ReadonlyArray<string>,
 		endpoints: EndpointHandlers,
+		chainID: Buffer,
```
Comment (shuse2) [inline framework/src/controller/channels/in_memory_channel.ts:41]: I think we can also add it to the ipc_channel

**Review by shuse2 — APPROVED** [review]

**Review by mitsuaki-u — APPROVED** [review]
Unit test is failing on jenkins

**Pull request merged**